app/models/user.rb

```diff
@@ -10,27 +10,28 @@ class User < ApplicationRecord
   has_secure_password
   validates :password, presence: true, length: { minimum: Settings.validates.password.minimum }
 
-  def User.digest(string)
-    cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST :
-      BCrypt::Engine.cost
-    BCrypt::Password.create(string, cost: cost)
-  end
+  class << self
+    def digest string
+      cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST :
+        BCrypt::Engine.cost
+      BCrypt::Password.create(string, cost: cost)
+    end
 
-  def User.new_token
-    SecureRandom.urlsafe_base64
-  end
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
 
-  def remember
-    self.remember_token = User.new_token
-    update_attribute(:remember_digest, User.digest(remember_token))
-  end
+    def remember
+      self.remember_token = User.new_token
+      update_attribute(:remember_digest, User.digest(remember_token))
+    end
 
-  def authenticated?(remember_token)
-    BCrypt::Password.new(remember_digest).is_password?(remember_token)
-  end
+    def authenticated?(remember_token)
+      BCrypt::Password.new(remember_digest).is_password?(remember_token)
+    end
 
-  def forget
-    update_attribute(:remember_digest, nil)
+    def forget
```

Comment:
It's an instance method. So remove this from the class method block

Reply:
OK Anh